templates/base.html.twig

```diff
@@ -15,13 +15,10 @@
             Those two blocks defines frontend entrypoint for CSS and JavaScript assets
             See https://symfony.com/doc/current/frontend.html
         #}
-        {% block stylesheets %}
-            {{ encore_entry_link_tags('app') }}
-        {% endblock %}
+        {% block stylesheets %}{% endblock %}
         {% block javascripts %}
-            {{ encore_entry_script_tags('app') }}
+            {{ importmap('app') }}
```

Comment:
I like the PR, as well as asset mapper that add a real value in sf world (your live was extremely valueable)
I am just challenging the full removal of « webpack encore »
Could it be wise, in the « migration » phase to showcase both ? (Dont know the faisability in demo repo)
To let devs choose their best case scenario / what  is best for their needs
Asset mapper is a new card added in the deck of devs and encore should remains discoverable in the meantime 
Wdyt ?

Reply:
If I understood Ryan well (@weaverryan can confirm) AssetMapper is the only future of Symfony asset handling and Webpack Encore is the past. Sooner than later, Encore will be marked as deprecated and AssetMapper will be the only option.

Reply:
That's exactly correct. I'm certainly not in a rush to deprecate Encore & I want to keep supporting the many apps using it. But AssetMapper is the future and that's what people should discover.

Reply:
oki copy that, thx for clarification both of you 👍🏻

Reply:
for context https://github.com/symfony/webpack-encore/pull/1239